packages/firestore/src/core/sync_engine.ts

```diff
@@ -787,10 +787,7 @@ export class SyncEngine implements RemoteSyncer, SharedClientStateSyncer {
     err: FirestoreError
   ): Promise<void> {
     if (isPrimaryLeaseLostError(err)) {
-      log.debug(
-        LOG_TAG,
-        'Unexpectedly lost primary lease, attempting to recover'
-      );
+      log.debug(LOG_TAG, 'Unexpectedly lost primary lease');
```

Comment:
You still need to update the method name and comment...  Please mention that when this error is thrown, there will automatically be a lease refresh scheduled so we should very quickly get an applyPrimaryState() call from the persistence layer.

Reply:
Update the method and comment. Method is now `ignoreIfPrimaryLeaseLoss`.